scripts/opentitan/verilate.py

```diff
@@ -497,25 +35,35 @@ def main():
         desc = modules[__name__].__doc__.split('.', 1)[0].strip()
         argparser = ArgumentParser(description=f'{desc}.')
         files = argparser.add_argument_group(title='Files')
-        files.add_argument('flash', nargs=1, type=FileType('rb'),
-                           metavar='ELF|VMEM', help='flash file')
+        files.add_argument('app', nargs='*', metavar='ELF',
+                            help='application file, may be repeated')
         files.add_argument('-V', '--verilator',
                            help='Verilator executable')
-        files.add_argument('-R', '--rom', type=FileType('rt'),
-                           metavar='VMEM', required=True,
-                           help='ROM file')
-        files.add_argument('-O', '--otp', type=FileType('rt'),
-                           metavar='VMEM',
+        files.add_argument('-R', '--rom', metavar='FILE', action='append',
+                           default=[],
+                           help='ROM file, may be repeated')
+        files.add_argument('-M', '--ram', metavar='FILE', action='append',
+                           default=[],
+                           help='RAM file, may be repeated')
+        files.add_argument('-F', '--flash', metavar='FILE', action='append',
+                           default=[],
+                           help='eFlash file, may be repeated')
+        files.add_argument('-O', '--otp', metavar='VMEM',
                            help='OTP file')
         files.add_argument('-K', '--keep-tmp', action='store_true',
                            help='do not automatically remove temporary files '
                                 'and dirs on exit')
         files.add_argument('-D', '--tmp-dir',
                            help='store the temporary files in the specified '
                                 'directory')
+        files.add_argument('-c', '--config', metavar='CFG',
+                           help='input QEMU OT config file '
+                                '(required w/ non scrambled ROM images')
```

Comment:
```suggestion
                                '(required w/ non scrambled ROM images)')
```
(repeated in the `verilate.md` docs later)

Reply:
fixed x2